build.sbt

```diff
@@ -31,7 +31,7 @@ libraryDependencies ++= Seq(
     exclude("org.bouncycastle", "bcprov-ext-jdk15on")
     exclude("org.bouncycastle", "bcutil-jdk15on")
     exclude("org.bouncycastle", "bcpkix-jdk15on"),
-  "org.broadinstitute.dsde.workbench" %% "workbench-notifications" % "0.3-a79c7f9"
+  "org.broadinstitute.dsde.workbench" %% "workbench-notifications" % "0.3-ac255960-SNAP"
```

Comment:
Will update to non-SNAP once https://github.com/broadinstitute/workbench-libs/pull/1229 is merged